.github/workflows/ci-docs.yml

```diff
@@ -1,4 +1,4 @@
-name: ci
+name: ci-docs
```

Comment:
The name needs to be "ci" -- see https://github.com/elastic/apm-server/pull/10002#discussion_r1065297736